packages/react-next/src/components/ExtendedPicker/BaseExtendedPicker.tsx

```diff
@@ -41,40 +38,18 @@ export class BaseExtendedPicker<T, P extends IBaseExtendedPickerProps<T>>
       // TODO: determine whether this can be removed
       // eslint-disable-next-line react/no-unused-state
```

Comment:
While you're here do you think it would be possible to remove this from state? The no-unused-state rule means the static analysis thinks the state member is never accessed (even if it's sometimes updated).

Reply:
I want to clean this up when I clean up the rest of the linting rules since it would be nice to batch those together. First get all of these changes and make it compliant, then remove all the anys and fix that so it can be done in stages.